src/main/webapp/results/results.js

```diff
@@ -0,0 +1,199 @@
+import React, { useState } from 'react'
+import { connect } from 'react-redux'
+import { Set } from 'immutable'
+import { getSelected, setSelected } from '../store/results'
+import { useKeyPressed } from '../react-hooks'
+
+import Paper from '@material-ui/core/Card'
+import Table from '@material-ui/core/Table'
+import Toolbar from '@material-ui/core/Toolbar'
+import Typography from '@material-ui/core/Typography'
+import TableHead from '@material-ui/core/TableHead'
+import TableRow from '@material-ui/core/TableRow'
+import TableCell from '@material-ui/core/TableCell'
+import TableBody from '@material-ui/core/TableBody'
+import More from '@material-ui/icons/UnfoldMore'
+import Less from '@material-ui/icons/UnfoldLess'
+import DefaultThumbnail from '../thumbnail/thumbnail'
+import Button from '@material-ui/core/Button'
+
+const cellStyles = {
+  minWidth: 150,
+  maxWidth: 450,
+}
+
+const ExpandButton = props => {
+  const { expanded, onClick } = props
+  return (
+    <Button
+      style={{
+        minWidth: 25,
+        marginLeft: 5,
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        justifyContent: 'center',
+      }}
+      onClick={onClick}
+    >
+      {expanded ? <Less /> : <More />}
+    </Button>
+  )
+}
+
+const Description = props => {
+  const [expanded, setExpanded] = useState(false)
+  const { text } = props
+  const long = text.length > 250
+
+  return (
+    <div
+      style={{
+        ...cellStyles,
+        display: 'flex',
+        width: '20vw',
+        alignItems: 'stretch',
+      }}
+    >
+      <Typography>
+        {!long || expanded ? text : text.substring(0, 100).concat('...')}
+      </Typography>
+      {long ? (
+        <ExpandButton
+          expanded={expanded}
+          onClick={e => {
+            e.stopPropagation()
+            setExpanded(!expanded)
+          }}
+        />
+      ) : null}
+    </div>
+  )
+}
+
+const getCellContent = (attribute, result, Thumbnail) => {
```

Comment:
How difficult would it be to make Thumbnail part of `result`?

Reply:
We're intentionally leaving it uncoupled so we can provide thumbnails however we want.